api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -601,6 +601,41 @@ message SessionConnect {
   ];
 }
 
+// FileTransferRequestEvent happens when a FileTransferRequest is created, updated, approved, or denied.
+message FileTransferRequestEvent {
+  // Metadata is a common event metadata
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // SessionMetadata is a common event session metadata
+  SessionMetadata Session = 2 [
```

Comment:
Don't we want to include the user metadata?

Reply:
We can but I don't see it being necessary at this point. The process doesn't really "care" who did it, as the approvers are stored in the request, as well as broadcasted (in the future PR) to the shared session on each action. This is mostly just an event to notify an updated happened to the state of the event.